docs/Exclude_Object.md

```diff
@@ -0,0 +1,106 @@
+# Exclude Obects
+
+The `[exclude_object]` module allows Klipper to exclude objects while a print is in progress.
```

Comment:
It would be preferable to wrap documentation files to 80 columns.

Also, new documents should be added to overview.md and to docs/_klipper3d/mkdocs.yml  (as mentioned in docs/CONTRIBUTING.md ).